acceptance/framework/k8s/helpers.go

```diff
@@ -0,0 +1,60 @@
+package k8s
```

Comment:
Great question! there is a reason: it's because putting these methods in the `helpers` package will create a circular dependency because they need to import `environment` and `config` packages, and those packages already import `helpers`

Reply:
Perfect!! that makes complete sense. Can we move some of the methods that are into `helpers/helpers.go` to this file? Here are a few that I thought were good candidates:
- func IsReady(pod corev1.Pod) bool
- func WaitForAllPodsToBeReady(t *testing.T, client kubernetes.Interface, namespace, podLabelSelector string)

Reply:
Done! Moved a few more methods out of that package into the `environment` package.

Reply:
thanks so much!! youre the best